source/wp-content/themes/wporg-news-2021/sass/page/front-page/_latest-release.scss

```
@@ -70,18 +71,27 @@ body.news-front-page .front__latest-release {
 		@include break-medium() {
 			flex-grow: 1;
 			font-size: 90px;
+			padding:
+				calc(var(--wp--custom--alignment--edge-spacing) / 2)
+				var(--wp--custom--alignment--edge-spacing)
+				var(--wp--custom--alignment--edge-spacing)
+				var(--wp--custom--alignment--edge-spacing);
+
+			&:hover {
```

You can add the background on focus too, with `focus-within`

```suggestion
			&:hover,
			&:focus-within {
```